wagtailio/areweheadlessyet/signal_handlers.py

```diff
@@ -0,0 +1,39 @@
+from threading import Thread
+
+import requests
+from django.conf import settings
+from wagtail.core.signals import page_published, page_unpublished
+
+from .models import AreWeHeadlessYetHomePage, AreWeHeadlessYetTopicPage
+
+
+def postpone(function):
+    """
+    Cheap aysnc, see https://stackoverflow.com/a/28913218
+    """
+
+    def decorator(*args, **kwargs):
+        t = Thread(target=function, args=args, kwargs=kwargs)
```

Comment:
If someone publishes/unpublishes a lot of pages at once, this could create a lot of threads. Could we use a thread-pool here to limit the number of threads that can be running?